### ISSUE TITLE
ln 66: uuid: command not found

### ISSUE THREAD

**Issue by cryptoB0T** [chbrown/overdrive#6]
getting an error when trying to use this

looks amazing! hope you can help. here's the debug log  below: 

do you know what this means?

```
~/D/b/odm ❯❯❯ overdrive download multipart.odm --verbose
+ printf 'Entering debug (verbose) mode\n'
Entering debug (verbose) mode
+ shift
+ [[ 0 -gt 0 ]]
+ [[ 1 -eq 0 ]]
+ [[ 1 -eq 0 ]]
+ HEADER_PRINTED=
+ for ODM in '"${MEDIA[@]}"'
+ for COMMAND in '"${COMMANDS[@]}"'
+ case $COMMAND in
+ download multipart.odm
+ license_path=multipart.odm.license
+ acquire_license multipart.odm multipart.odm.license
+ [[ -e multipart.odm.license ]]
++ uuid
++ tr /a-z/ /A-Z/
/usr/local/bin/overdrive: line 66: uuid: command not found
+ ClientID=
+ printf 'Generating random ClientID=%s\n' ''
Generating random ClientID=
++ xmlstarlet sel -t -v /OverDriveMedia/License/AcquisitionUrl multipart.odm
/usr/local/bin/overdrive: line 70: xmlstarlet: command not found
+ AcquisitionUrl=
```

**Comment by rok1024** [comment]
See **Prerequisites** section in [here](https://github.com/chbrown/overdrive/blob/master/README.md).

You need to have these auxiliary programs installed first.

**Comment by chbrown** [comment]
Hi @cryptoB0T, please see the workaround posted in #7, which I have now implemented in master (f84926d). 
Just [reinstall](https://github.com/chbrown/overdrive#install) and you shouldn't have to worry, at least not about that particular dependency :)
